src/bin/stratisd.rs

```diff
@@ -83,11 +83,20 @@ fn handle_udev_add(event: &libudev::Event) -> Option<(Device, PathBuf)> {
 /// To ensure only one instance of stratisd runs at a time, acquire an
 /// exclusive lock. Return an error if lock attempt fails.
 fn trylock_pid_file() -> StratisResult<File> {
-    let mut f = OpenOptions::new()
+    let mut f = match OpenOptions::new()
         .read(true)
         .write(true)
         .create(true)
-        .open(STRATISD_PID_PATH)?;
+        .open(STRATISD_PID_PATH)
+    {
+        Ok(f) => f,
+        Err(e) => {
+            if e.kind() == ErrorKind::PermissionDenied {
+                eprintln!("Please make sure to run stratisd as root.");
```

Comment:
Please just return the string in the error, as:

```return Err(StratisError::Error("Must be running as root in order to start daemon."));```

Reply:
Sorry, I tried that with `Into` trait before.
Should have better checked `StratisError`.